wp-content/themes/thinkery/style.css

```diff
@@ -1485,23 +1486,68 @@ p.entry-meta {
 	margin-bottom: 0;
 }
 
+.footer-widgets div ul li a {
+	color: white;
+	font-family: museo-sans, sans-serif;
+	font-size: .85em;
+	font-weight: 400;
+	text-decoration: none;
+}
+.footer-widgets #fa-icons {
```

Comment:
Lets try not to use IDs for styles, try to stay as general / non-specific as possible to avoid locking ourselves in down the line.

Reply:
Roger that, I was having issues getting the icons to respond properly to the 

```
display: flex;
justify-content: space-between;
```

properties without the additonal Div with the ID. Should I be steering towards an alternate method for achieving this?

Reply:
nevermind, i was able to hit this by specifically targeting the `<div> `wrapping the Icons with this

```
.footer-widgets div div section div div div {
display: flex;
justify-content: space-between;
}
```

I also considered giving the div class that lets us know the container is using `flex` and defining the css rules through that class, and may end up using that in the end depending on further review